scripts/release.helpers.js

```diff
@@ -0,0 +1,116 @@
+const readline = require('node:readline/promises');
+
+const { exec } = require('child-process-promise');
+
+async function checkGitStatus() {
+  console.log('Checking your PR branch...');
+  const resultBranch = await runExec('git branch --show-current', {
+    silent: true,
+  });
+  const branchName = resultBranch.stdout.toString().trim();
+  if (branchName === 'main') {
+    console.error(`🟠 You are at "main". Are you sure you wanna release a dev version here?`);
```

Comment:
lots of good defensive programming here that we need

Reply:
This PR is the biggest proof I'm human. Each `if` was a mistake I've done. 😅